### ISSUE TITLE
Unable to delete terminal text + terminals database is inaccessible error

### ISSUE THREAD

**Issue by geoffmcc** [gpakosz/.tmux#603]
Hello. I've been using these dot's for some time and love them. Unfortunately I believe an update in tmux caused some issues. I have the tmux dots on my macOS Monterey 12.5.1, Ubuntu 22.04 Laptop & Ubuntu 22.04 Raspberry Pi. The only one that gives me issues is on the macOS and it just started.

I am unable to delete any text in terminal once tmux is launched. If I try, the cursor moves forwards rather than backwards. Interestingly if I make a mistake 3 characters in, when I press delete it will only respond  3 times, but again rather than delete it goes forwards. Also, if I try to open nano I get an "error terminals database is inaccessible"

macOS - tmux 3.3a
Ubuntu 22.04 Laptop - tmux 3.2a
Ubuntu 22.04 Raspberry Pi - tmux 3.1c

So it seems to be just in tmux 3.3a. Attached is an example of me trying to delete text as well as the error when attempt to launch nano

https://user-images.githubusercontent.com/17417840/189445973-0c2ce986-1a96-4707-8d6d-04878e61efcc.mov

**Comment by gpakosz** [comment]
Hello @geoffmcc 👋

Does the problem happen when you launch tmux with
```
$ tmux -L test -f /dev/null
```
?

**Comment by geoffmcc** [comment]
Thanks @gpakosz for the quick reply.

No. As far as I can tell, everything is working fine when I launch like that.

**Comment by gpakosz** [comment]
Outside of tmux, what's the result of `$ infocmp -x tmux-256color` please?

**Comment by geoffmcc** [comment]
Although I'm not sure what it was, it must have been something I did on my end that caused the issue. I ended up doing a reset on my phone and laptop early this am and now after getting everything back the way I like it, tmux seems to be working fine now. Sorry to have wasted your time. It just really seemed to be a version issue to me at the time.

Just to answer though, what I am now getting outside of tmux 

`infocmp: couldn't open terminfo file (null).`
